Extensions/CodeGeneration/CodeGenerationDisplayDriver.cs

```diff
@@ -13,10 +13,14 @@ namespace Lombiq.HelpfulExtensions.Extensions.CodeGeneration
 {
     public class CodeGenerationDisplayDriver : ContentTypeDefinitionDisplayDriver

```

Comment:
Kellene devből merge a preview branchbe, majd ide, mert ez a class pont változott.

Note to self: [itt](https://localhost:44316/Admin/ContentTypes/Edit/HtmlWidget) met [itt](https://localhost:44316/Admin/ContentTypes/Edit/KeyDocument) meg [itt](https://localhost:44316/Admin/ContentTypes/Edit/Page) kell letesztelni.

Reply:
done